pkg/osbuild/bootupd_stage.go

```diff
@@ -107,6 +107,20 @@ func genMountsForBootupd(source string, pt *disk.PartitionTable) ([]Mount, error
 				mount.Partition = common.ToPtr(idx + 1)
 				mounts = append(mounts, *mount)
 			}
+		case *disk.LVMVolumeGroup:
+			for i := range payload.LogicalVolumes {
+				lv := &payload.LogicalVolumes[i]
+				mountable, ok := lv.Payload.(disk.Mountable)
+				if !ok {
```

Comment:
I suggest making this an error to prevent surprises if someone actually puts a non-mountable payload into an LV.